RobotMbed/src/main/java/de/fhg/iais/roberta/worker/MbedValidatorAndCollectorWorker.java

```diff
@@ -13,16 +14,20 @@
 public abstract class MbedValidatorAndCollectorWorker extends AbstractValidatorAndCollectorWorker {
```

Comment:
upper case names are misleading. They are used for static finals, but here they are regular mebers.
FREE_PINS -> freePins etc.